src/UserNamespaces.jl

```diff
@@ -73,15 +73,13 @@ end
 
 function check_kernel_version(;verbose::Bool = false)
     # Don't bother to do anything on non-Linux
-    if !Sys.islinux()
-        return true
-    end
+    Sys.islinux() || return false
```

Comment:
I personally don't really like the `condition || return false` idiom.

Reply:
Done!